netpyne_ui/templates/batch.py

```diff
@@ -9,8 +9,13 @@
 from netpyne import specs
 from netpyne.batch import Batch
 
-os.chdir(os.path.dirname(__file__))
+CURRENT_DIR = "."
+try:
+    CURRENT_DIR = os.path.dirname(__file__)
```

Comment:
In a general way, should we drop the use of `os.path` and use `pathlib` instead? It's perhaps really personal, but I find `pathlib` way more confortable and easy to use :D

Reply:
I never used pathlib honestly and I'm happy to learn about it :) In general I would not really question this matter of taste kind of code solution in PRs unless there is a deprecation matter or other reasons like efficiency

Reply:
Sorry if it was out of scope. I mentioned it because I saw the use of both libraries in the PR, so I wasn't sure which one was preferred or recommended, or if we had some guidelines in general. I apologize if it sounded out of bounds.

Reply:
Good point, I'm always happy to learn new things and I appreciate feedback of this kind. Let's just make sure it's clear it's a general advice especially when less experienced people are submitting the pr because the next thing you see is someone spending hours replacing one lib with the other spending hours and potentially introducing bugs because they may not be familiar with the other library